app/templates/views/guidance/index.html

```diff
@@ -84,6 +84,9 @@ <h4>Example</h4>
   <p>To personalize with the recipient's first name and include a reference number:</p>
   <p class="padding-2 bg-base-lightest">State WIC: Hello ((first name)), your reference is ((ref number)). Please provide this number when you call 123-123-1234 to make an appointment.</p>
```

Comment:
I don't know if 123 is a legit area code, but should we do '555' to guarantee the invalidity of the number?

Reply:
123-555-1234 or 123-555-5555 or 555-555-5555 etc.